cairo/tests/utils/strategies.py

```diff
@@ -328,12 +328,15 @@ def tuple_strategy(thing):
                 {address: st.from_type(Account) for address in addresses}
             ),
         ),
+        # Storage tries are not always present for existing accounts
+        # Thus we generate a subset of addresses from the existing accounts
         _storage_tries=st.fixed_dictionaries(
             {
                 address: trie_strategy(Trie[Bytes32, U256]).filter(
                     lambda t: bool(t._data)
                 )
-                for address in addresses
+                # TODO: change to a random number between 0 and len(addresses)
```

Comment:
why a todo ?
you can use a flatmap with st.integers(max_value=len(address)//2) or am I missing something ?

Reply:
thanks